app/models/api/fake_data/activity_report.rb

```diff
@@ -26,9 +26,15 @@ def data
         topics: [phrase, phrase]
       },
       links: {
-        self: "https://example.com/TODO",
-        html: "https://example.com/TODO"
+        self: "https://ttahub.ohs.acf.hhs.gov/api/v1/activity-reports/display/#{display_id}",
```

Comment:
I think they do? It felt weird writing a test against `example.com` so that's why I changed it. In the end, I don't think it matters much either way. You'll either get a broken link to example.com or a broken link to the ttahub.